src/views/Checkout.vue

```diff
@@ -155,13 +155,13 @@ export default class Checkout extends Vue {
         const minHeight = 890; // Height at which two lines fit at bottom, also if logos over carousel shown.
         this.screenFitsDisclaimer = window.innerWidth >= minWidth && window.innerHeight >= minHeight;
     }
+}
 
-    private data() {
-        return {
-            Currency,
-        };
-    }
+namespace Checkout {
+    export const Currency = PublicCurrency;
```

Comment:
I have a slight preference for adding `export type Currency = PublicCurrency` here and using `Checkout.Currency` instead of `PublicCurrency` in the type annotations.
But if you prefer the way you did it, just keep it that way.